src/deepsparse/transformers/__init__.py

```diff
@@ -33,11 +33,22 @@
 
 
 _LOGGER = _logging.getLogger(__name__)
-_NM_TRANSFORMERS_TAR_TEMPLATE = (
-    "https://github.com/neuralmagic/transformers/releases/download/"
-    "{version}/transformers-4.7.0.dev0-py3-none-any.whl"
-)
-_NM_TRANSFORMERS_NIGHTLY = _NM_TRANSFORMERS_TAR_TEMPLATE.format(version="nightly")
+
+
+def _get_transformers_install_url(version):
+    if version == "nightly":
+        transformers_base_version = "4.18.0"
+        release = "nightly"
+    else:
+        transformers_base_version = "4.18.0" if version >= "0.12" else "4.7.0"
```

Comment:
since this is on main isn't it only affecting >=0.12?

Reply:
good point